src/index.html

```diff
@@ -1,12 +1,405 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect"
+          href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+          href="https://fonts.gstatic.com"
+          crossorigin="anonimus">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+
+    <header class="header">
+      <a href="index.html" class="header__logo">
+        <img src="images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#" >Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#" >Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#" >Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#" >Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#" >Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <div class="card catalog__item" data-qa="card">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="imac_image"
+        >
+
+        <div class="card__title title">
+          <p class="title__text">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+        </div>
+
+        <div class="card__code code">
```

Comment:
```suggestion
        <div class="card__code">
```
you don`t need to add own block to each element,